### ISSUE TITLE
error using pip/ setup.py command?

### ISSUE THREAD

**Issue by rohan11103** [InstaPy/InstaPy#1102]
## Expected Behavior

New to coding but I read that the pip command should install a python package. I tried both ways  listed in the getting started but neither have worked. 

## Current Behavior
```
Geetanjalis-iMac-462:~ geetanjalisharma$ cd Instapy
Geetanjalis-iMac-462:Instapy geetanjalisharma$ pip install .
-bash: pip: command not found
Geetanjalis-iMac-462:Instapy geetanjalisharma$  python setup.py install
python: can't open file 'setup.py': [Errno 2] No such file or directory
Geetanjalis-iMac-462:Instapy geetanjalisharma$
```

## Possible Solution (optional)
Unknown

## InstaPy configuration
```python
insta_username = '----(my username)'
insta_password = '(my password)'

# if you want to run this script on a server,
# simply add nogui=True to the InstaPy() constructor
session = InstaPy(username=insta_username, password=insta_password)
session.login()

# set up all the settings
session.set_upper_follower_count(limit=2500)
session.set_do_comment(True, percentage=10)
session.set_comments(['aMEIzing!', 'So much fun!!', 'Nicey!'])
session.set_dont_include(['friend1', 'friend2', 'friend3'])
session.set_dont_like(['pizza', 'girl'])

# do the actual liking
session.like_by_tags(['natgeo', 'world'], amount=100)

# end the bot session
session.end()
```

**Comment by converge** [comment]
hey buddy 

try it:
```sudo easy_install pip```

**Comment by rohan11103** [comment]
Last login: Sun Dec 17 11:59:04 on ttys000
Geetanjalis-iMac-462:~ geetanjalisharma$ cd Instapy
Geetanjalis-iMac-462:Instapy geetanjalisharma$ sudo easy_install pip
Password:
Searching for pip
Best match: pip 9.0.1
Processing pip-9.0.1-py2.7.egg
pip 9.0.1 is already the active version in easy-install.pth
Installing pip script to /usr/local/bin
Installing pip2.7 script to /usr/local/bin
Installing pip2 script to /usr/local/bin

Using /Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg
Processing dependencies for pip
Finished processing dependencies for pip
Geetanjalis-iMac-462:Instapy geetanjalisharma$ python setup.py install
python: can't open file 'setup.py': [Errno 2] No such file or directory
Geetanjalis-iMac-462:Instapy geetanjalisharma$ python quickstart.py
python: can't open file 'quickstart.py': [Errno 2] No such file or directory
Geetanjalis-iMac-462:Instapy geetanjalisharma$

**Comment by rohan11103** [comment]
I think I'm still doing something wrong :(

**Comment by converge** [comment]
sounds like you dont have setup.py
try it: https://www.youtube.com/watch?v=2Ci-hXU1IEY
It's almost the same for MacOS

**Comment by rohan11103** [comment]
Hey! So I watched the video, and I got this far. 
Last login: Sun Dec 17 13:11:14 on ttys000
Geetanjalis-iMac-462:~ geetanjalisharma$ pwd
/Users/geetanjalisharma
Geetanjalis-iMac-462:~ geetanjalisharma$ git clone https://github.com/timgrossmann/InstaPy.git
fatal: destination path 'InstaPy' already exists and is not an empty directory.
Geetanjalis-iMac-462:~ geetanjalisharma$ cd InstaPy/
Geetanjalis-iMac-462:InstaPy geetanjalisharma$ ls -l
total 8
-rw-r--r--  1 geetanjalisharma  staff  965 Dec 16 19:16 docker_quickstart.py
Geetanjalis-iMac-462:InstaPy geetanjalisharma$



But then you say switch to the quickstart.py and start editing the program (that I was previously using atom to do so), but Idk how you do that in the terminal window. Whats that step? Also the terminal window you are using looks different, Im running iterm2, is that ok?
